Support/Fuego/Mechanism/Models/hydrogen_cyanide/make-mechanism.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
can we make this `make-mechanism.sh` look like the others? And then just move the *.inp and *.dat into this folder (instead of the `orig` folder)?